app/views/templates/new_import.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= page_title "Import Template" %>
+
+<div class="max-w-2xl">
+  <h1 class="text-xl mb-4">Import Template from Archive</h1>
+
+  <p class="mb-4 text-gray-700">
+    Upload a ZIP file containing a template XML descriptor and image files.
+    Compatible with Concerto 1 and Concerto 2 template archives.
+  </p>
+
+  <%= form_with url: create_import_templates_path, multipart: true, class: "space-y-4" do |form| %>
+    <div>
+      <%= form.label :archive, "Template Archive (ZIP)", class: "block font-medium mb-2" %>
+      <%= form.file_field :archive, accept: ".zip", required: true, class: "block w-full text-sm text-gray-500 file:mr-4 file:py-2 file:px-4 file:rounded-md file:border-0 file:text-sm file:font-semibold file:bg-blue-50 file:text-blue-700 hover:file:bg-blue-100" %>
+      <p class="mt-1 text-sm text-gray-500">
+        Select a .zip file containing template.xml and image files
+      </p>
+    </div>
+
+    <div class="flex gap-2">
+      <%= form.submit "Import Template", class: "btn" %>
+      <%= link_to "Cancel", templates_path, class: "btn btn-secondary" %>
+    </div>
+  <% end %>
+
+  <div class="mt-8 p-4 bg-gray-50 rounded-md">
```

Comment:
Remove mention of the sample templates here. That's not needed, users can find their own templates.